Readme.md

```diff
@@ -0,0 +1,181 @@
+# Portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Images\html_css.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Abiy Capston One Readme</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ABIY-PORTFOLIO] <a name="about-project"></a>
+
+**[Abiy-Portfolio]** This is a capstone created during my Microverse course. 
+The capstone is able to function (responsive to) both on the mobile and desktop version.
+
```

Comment:
- Hey, let's also update the name to the current project you've used as this is not the current name. this is a capstone project, so the name shouldn't be your portfolio.